config/task.config

```diff
@@ -0,0 +1,54 @@
+process {
+
+    withName: Cutadapt_1 {
+        ext.prefix = {"${meta.id}_1"}
+        container='quay.io/biocontainers/cutadapt:4.6--py39hf95cd2a_1'
+        ext.args ={" -a $params.linked_adapter_1 --discard-untrimmed "}
+    }
+
+     withName: Cutadapt_2 {
```

Comment:
Cutadapt can process multiplle adapters in the same command.

Reply:
This needs to be done, because linked adapters are needed to remove the primers and locus specific primers. Using the revcomp option of cutadapt adjust the read direction 5' for all reads, losing the information. For this reason two cutadapt processes are required.